docs/integration/Sql.md

```diff
@@ -19,7 +19,7 @@ It supports:
 
 ## Configuration
 
-You have to configure database connection pool you will be using in your `Sql` enricher.
+You have to configure database connection pool you will be using in your `Sql` enricher. It is performed at the root level of the configuration file (the same level as all other configs from Designer configuration area) - see extract of the [configuration file](../installation_configuration_guide/Common.md#configuration-areas).
```

Comment:
1. "configure database connection pool" -> "configure a database connection pool"
2. "see extract of the" -> the same as in one of my previous comments - it'd be better to have a direct link to the config file section